docs-src/content/_index.md

```diff
@@ -2,11 +2,226 @@
 title: "Cubic SDK Documentation"
 ---
 
-# Cubic SDK
+# Cubic API Overview
 
-Cubic is Cobalt's speech recognition engine.  It can be bundled into your
-application as a library, or deployed on-prem and accessed over the network.
-This documentation refers to accessing the network-based Cubic server.
+Cubic is Cobalt’s automatic speech recognition (ASR) engine. It can be deployed on-prem and accessed over the network or on your local machine via an API. We currently support Go, Python and Java, and are adding support for more languages.
+
+Once running, Cubic’s API provides a method to which you can stream audio. This audio can either be from a microphone or a file. We recommend uncompressed WAV as the encoding, but support other formats such as MP3.
+
+![Cubic setup](cubicoverview.png)
+
+Cubic’s API provides a number of options for returning the speech recognition results. The results are passed back using Google’s protobuf library, allowing them to be handled natively by your application. Cubic can estimate its confidence in the transcription result at the word or utterance level, along with timestamps of the words. Confidence scores are in the range 0-1. Cubic’s output options are described below.
+
+## Automatic Transcription Results
+
+The simplest result that Cubic returns is its best guess at the transcription of your audio. Cubic recognizes the audio you are streaming, listens for the end of each utterance, and returns the speech recognition result. 
+
+Cubic maintains its transcriptions in an N-best list, i.e. is the top N transcriptions from the recogniser. The best ASR result is the first entry in this list. A json representation of Cubic’s N-best list with utterance-level confidence scores is:
+
+```
+{
+  "alternatives": [
+    {
+      "transcript": "TOMORROW IS A NEW DAY",
+      "confidence": 0.514
+    },
+    {
+      "transcript": "TOMORROW IS NEW DAY",
+      "confidence": 0.201
+    },
+    {
+      "transcript": "TOMORROW IS A <UNK> DAY",
+      "confidence": 0.105
+    },
+    {
+      "transcript": "TOMORROW IS ISN'T NEW DAY",
+      "confidence": 0.093
+    },
+    {
+      "transcript": "TOMORROW IS A YOUR DAY",
+      "confidence": 0.087
+    }
+  ],
+}
+```
+
+A single stream may consist of multiple utterances separated by silence. Cubic handles each utterance separately. For longer utterances, it is often useful to see the partial speech recognition results while the audio is being streamed. For example, this allows you to see what the ASR system is predicting in real-time while someone is speaking. Cubic supports both partial and final ASR results.
```

Comment:
This may be petty, but I would suggest putting this into two paragraphs

```txt
A single stream may consist of multiple utterances separated by silence. Cubic handles each utterance separately. 

For longer utterances, it is often useful to see the partial speech recognition results while the audio is being streamed. For example, this allows you to see what the ASR system is predicting in real-time while someone is speaking. Cubic supports both partial and final ASR results.
```

Reply:
done